translations/fr.json

```diff
@@ -698,17 +698,17 @@
     "gamestatus.cities": "{0} villes",
     "gamestatus.destroyed": "détruite(s)",
     "gamestatus.tasks": "Tâches {0}/{1}",
-    "gamestatus.tasks.complete": "Complétée!",
+    "gamestatus.tasks.complete": "Compl\00E9t\00E9e!",
```

Comment:
This line caused an error. If you want to use special Unicode characters do it either directly or via Unicode format: `Compl\u00E9t\u00E9e!` instead of `Compl\00E9t\00E9e!`